unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/DCLVideoTexture.cs

```diff
@@ -17,6 +18,7 @@ public class DCLVideoTexture : DCLTexture
 #endif
 
         private const float OUTOFSCENE_TEX_UPDATE_INTERVAL = 1.5f;
+        private const float VIDEO_PROGRESS_UPDATE_INTERVAL = 1f;
```

Comment:
Can we specify the measure of time for this? I guess is in seconds, so something like SECONDS_VIDEO_PROGRESS_UPDATE_INTERVAL

Reply:
I like this, made the change on last commit